config/vufind/searches.ini

```diff
@@ -254,10 +254,14 @@ WorkKeys = year
 ;
 ; Available modules recommended for use in the side area:
 ;
-; CatalogResults:[GET parameter]:[result limit]
+; CatalogResults:[GET parameter]:[result limit]:[hidden filters key]:[custom heading]
```

Comment:
[hidden filters key] would work in any AbstractSearchObject recommendation module.  [custom heading] would also if I update the templates.  I don't know how far to go documenting that for each of these recommendation modules.  Every AbstractSearchObject recommendation module already has a lot of duplicate explanation (which makes sense since  they're near-identical implementations).

Reply:
I added a TODO

Reply:
I might also refactor the various AbstractSearchObject modules somehow so this explanation can be in a single place under side modules.